api.ts

```diff
@@ -2193,14 +2206,14 @@ export interface SyncTableOptions<
    * This should describe the entities being synced. For example, a sync table that syncs products
    * from an e-commerce platform should be called 'Products'. This name must not contain spaces.
    *
-   * Important: This value acts as a unique ID for the table, and updating it later is a breaking change. 
+   * Important: This value acts as a unique ID for the table, and updating it later is a breaking change.
```

Comment:
This _was_ a change made by Faraz for the display name change - I guess we have different trailing space settings in our IDEs

Reply:
I guess we fixed the linter for the `packs` repo to enforce a common `prettier` style via the CI, but didn't do that work for the SDK. We should do that at some point.